feature-group-definitions/svg2-script-html-equivalence.yml

```
@@ -1,3 +1,4 @@
+name: "SVG <script>: async & defer"
```

Actually no `:` I think:

```suggestion
name: "SVG <script> async & defer"
```